src/layouts/Layout.astro

```diff
@@ -57,6 +60,7 @@ const tabs = {
   <body
     class={clsx(
       'flex h-full sm:min-h-screen flex-col bg-white text-neutral-dark mx-auto',
+      `primary-text-${primaryText}`,
```

Comment:
Could we use the conditional class names here rather than constructing the class name?

```
{ "primary-text-white": !useBlackText },
{ "primary-text-black: useBlackText }
```